plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/features/core/model/ModelSubmissionHandler.java

```diff
@@ -55,28 +56,32 @@
     * </p>
     *
     * @param modelState The model state to submit.
+    * @param reason     The action/operation that caused the model update
```

Comment:
```suggestion
    * @param reason     The optional reason that caused the model update.
```
I believe this is still out-dated.